pkg/querier/querier.go

```diff
@@ -167,7 +167,7 @@ func New(cfg Config, limits *validation.Overrides, distributor Distributor, stor
 	})
 
 	engine := promql.NewEngine(promql.EngineOpts{
-		Logger:             util_log.Logger,
+		Logger:             log.NewNopLogger(),
```

Comment:
Why noop logger here? It's not a test.

Reply:
Wow, good catch. I must have accidentally ran the test find and replace code across the package and missed it. Thanks for spotting this. It would have been a major regression.

Will fix this up ASAP.